oak-commons/src/main/java/org/apache/jackrabbit/oak/commons/sort/ExternalSort.java

```diff
@@ -460,11 +603,20 @@ public static <T> File sortAndSave(List<T> tmplist,
      * @param typeToString
      *        function to map string to custom type. User for coverting line to custom type for the
      *        purpose of sorting
+     * @param filterPredicate
```

Comment:
Can we clarify this to say that (filter out comes out as removing to me but here we negate the keep predicate)
`predicate to keep relevant data which need to be sorted`